pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientBuilderImpl.java

```diff
@@ -164,6 +167,7 @@ public ClientBuilder listenerThreads(int numListenerThreads) {
 
     @Override
     public ClientBuilder connectionsPerBroker(int connectionsPerBroker) {
+        checkArgument(connectionsPerBroker >= 0, "connectionsPerBroker needs to be >= 0");
```

Comment:
Per the java doc on the `ClientBuilder` interface, the value must be greater than 0.
```suggestion
        checkArgument(connectionsPerBroker > 0, "connectionsPerBroker needs to be > 0");
```

Reply:
done